src/main.cpp

```diff
@@ -669,51 +691,57 @@ int main(int argc, char* argv[])
 		return 0;
 	}
 
-	if (cli.command == "version" || cli.command == "--version" || cli.command == "-version" || cli.command == "-v") {
+	if (cli.command == "version" || cli.command == "--version" || cli.command == "-version") {
 		logf(g_version_string);
 		return 0;
 	}
 
-	if (argc == 2) {
-		start_viewer(argv[1]);
+	if (cli.command == "editor" || cli.command == "empty") {
+		start_viewer(std::string());
+		return 0;
 	}

```

Comment:
how is this useful?

Reply:
For opening map from editor instead of command line